google/cloud/pubsub/benchmarks/throughput.cc

```diff
@@ -248,6 +265,11 @@ class PublishWorker {
             OnAck();
           });
       ++send_count;
+      if (enable_pacing && i != 0 && i % kPacingCount == 0) {
```

Comment:
Nit: The asymmetry in cases like these always strikes me.  How about `(i + 1) % kPacingCount == 0`?

Reply:
I like it, thanks.